moveit_planners/pilz_industrial_motion_planner/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ find_package(tf2_ros REQUIRED)
 find_package(orocos_kdl REQUIRED)
 find_package(Boost REQUIRED COMPONENTS)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  -std=c++11")
```

Comment:
We don't want to use c++11 but a more modern version (Right now c++17). Why is this change necessary for support ob bullseye?